openid-connect-4-credential-issuance-1_0.md

```diff
@@ -382,9 +457,84 @@ Location: https://client.example.net/cb?
+
+## Pushed Authorization Endpoint
+
+This step is OPTIONAL. Authorization requests in this specification follow the definition given in Section 3.1.2 of [@!OpenID], with the exception of the differences specified in the Simple Issuance Flow of this document. In case of complex flows or when the size of the request is large due to the query language or encrypting, clients are RECOMMENDED to use pushed authorization requests.
+
+### Pushed Authorization Request
+
+Authorization Requests are made as defined in Section 3.1.2.1 of [@!OpenID]. This specification defines the following requirements beyond [@!OpenID]:
+
+The authorization request MUST include the `claims` parameter defined in section 5.5 of [@!OpenID] with a new top-level element `credentials`. 
+
+* `credentials`: REQUIRED. JSON array where every object describes a credential being requested and (optionally) may contain references to verifiable 
+presentations provided as pre-requisite for credential issuance. The objects have the following structure:
+
+  * `manifest_id`: CONDITIONAL. JSON String refering to a credentoal manifest published by the credential issuer. `manifest_id` and `type` are mutual exclusive. 
+  * `presentation_submission`: CONDITIONAL. JSON object as defined in [@DIF.CredentialManifest]. This object refers to verifiable presentations required for the respective credential accoridng to the credential manifest and provided in an authorization request. All entries in the `descriptor_map` refer to verifiable presentations provided in the `vp_token` authorization request parameter.  
+
+Below is a non-normative example of a `claims` parameter with `presentation_submission`:
+
+```json=
+{
+   "credentials":[
+      {
+         "manifest_id":"WA-DL-CLASS-A",
+         "format":"ldp_vc",
+         "presentation_submission":{
+            "id":"a30e3b91-fb77-4d22-95fa-871689c322e2",
+            "definition_id":"32f54163-7166-48f1-93d8-ff217bdb0653",
+            "descriptor_map":[
+               {
+                  "id":"input_1",
+                  "format":"jwt_vc",
+                  "path":"$.verifiableCredential[0]"
+               }
+            ]
+         }
+      }
+   ]
+}
+```
+
+The authorization request MAY contain a parameter `vp_token` as defined in [@OIDC4VP] used to convey required verifiable presentations. The verifiable presentations passed in this parameter MUST be bound to a `p_nonce` generated by the respective issuer and the issuer`s identifier. 
+
+POST /op/par HTTP/1.1
+    Host: as.example.com
+    Content-Type: application/x-www-form-urlencoded
+
+    &response_type=code
+    &client_id=CLIENT1234
+    &nonce=duk681S8n00GsJpe7n9boxdzen
+    &redirect_uri=https%3A%2F%2Fclient.example.org%2Fcb
+    &scope=openid
+    &claims=...
+    &vp_token=...
+
+### Pushed Authorization Response
+
+The Pushed Authorization Response follows the definition in [@!RFC9126].
+
 ## Token Endpoint
```

Comment:
purpose - exchange code to the access_token
precondition/dependency - successful response from the Authz req.
endpoint description
outcome - getting access_token